test/feature-benchmark/scenarios.py

```diff
@@ -572,6 +572,52 @@ def benchmark(self) -> MeasurementSource:
         )
 
 
+class AccumulateReductions(Dataflow):
+    """Benchmark the accumulation of reductions."""
+
+    def before(self) -> Action:
+        return TdAction(
+            """
+> DROP TABLE IF EXISTS t CASCADE;
+> CREATE TABLE t (a int, b int, c int, d int);
+
+> CREATE MATERIALIZED VIEW data AS
+  SELECT a, a AS b FROM generate_series(1, 10000000) AS a
+  UNION ALL
+  SELECT a, b FROM t;
+
+> INSERT INTO t (a, b) VALUES (1, 1);
+> INSERT INTO t (a, b) VALUES (0, 0);
+
+> DROP CLUSTER IF EXISTS idx_cluster CASCADE;
+> CREATE CLUSTER idx_cluster SIZE '1-8G', REPLICATION FACTOR 1;
+
+> CREATE VIEW accumulable AS
+  SELECT
+    a,
+    sum(a) AS sum_a, count(a) as cnt_a,
+    sum(b) AS sum_b, count(b) as cnt_b
+  FROM data
+  GROUP BY a;
+"""
+        )
+
+    def benchmark(self) -> MeasurementSource:
+        return Td(
+            """
+> SELECT 1
+  /* A */
+1
+
+> CREATE INDEX i_accumulable IN CLUSTER idx_cluster ON accumulable(a);
+
+> SELECT count(*) FROM accumulable;
+  /* B */
```

Comment:
Similarly to the previous comment, it is also ideal to `SET CLUSTER = idx_cluster` here prior to the `SELECT`. You can refer to some similar code in `HydrateIndex(Scenario)`.